### PR TITLE
chore(docs): update readme

### PR DESCRIPTION
**Pull request by Arqu**
## Description

Is the tagline correct?
Can we enable this discord widget?

## Change checklist

- [ ] Self-review.
- [ ] Documentation updates following the [style guide](https://rust-lang.github.io/rfcs/1574-more-api-documentation-conventions.html#appendix-a-full-conventions-text), if relevant.
- [ ] Tests if relevant.
- [ ] All breaking changes documented.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,16 @@
...
-  <!-- docs.rs docs -->
-  <a href="https://docs.rs/iroh">
-    <img src="https://img.shields.io/badge/docs-latest-blue.svg?style=flat-square"
-      alt="docs.rs docs" />
-  </a>
-</div>
+<h1 align="center"><a href="https://iroh.computer"><img alt="iroh" src="./.img/iroh_wordmark.svg" width="100" /></a></h1>
+
+<h3 align="center">
+Build distributed apps that raise the status quo for your users.
```
Comment (dignifiedquire) [inline README.md:4]:
```
A toolkit for building distributed applications
```
should be the tagline

**Comment by dignifiedquire** [comment]
Screenshot 2024-07-05 at 17 29 15

widget is enabled

File: README.md [hunk 2]
```diff
@@ -38,9 +30,17 @@ Bytes, Distributed.
 </div>
 <br/>
 
+## Features
```
Comment (Arqu) [inline README.md:33]: Should we expand this with the full set of items instead of just the 3 layers? What should we include here?

Comment (dignifiedquire) [reply README.md:33]: probably, but lets get the existing improvements merged now, and do that in a follow up

File: README.md [hunk 3]
```diff
@@ -38,9 +30,17 @@ Bytes, Distributed.
...
+
+- Documents - Authors create and join documents: mutable key-value stores that multiple users read from, write to, and sync with, subscribing to live updates in real time.
+- Blobs - Iroh works with content-addressed blobs of opaque data, which are often the bytes of a file.
+- Networking - At the core of iroh is the ability to connect any two devices, no matter where they are.
```
[the remark this answers is not in the document]
Comment (Arqu) [reply README.md:37]: I like it, on the other hand, the above is taken directly from the iroh.computer site. Should we do it separately and try to keep it consistent?

**Review by dignifiedquire — APPROVED** [review]

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (matheus23) [reply README.md:37]: Good point. Yeah I'll open a PR for the website, and we can discuss with @b5 , too :)

**Pull request merged**